### PR TITLE
fixed #18558: Crash when trying to open 2 cloud scores at the same time

### LINKED ISSUE #18558: Crash when trying to open 2 cloud scores at the same time
### Issue type

Crash or freeze

### Bug description

When opening a cloud score while another one is being (down)loaded, the program crashes

### Steps to reproduce

1. Open the 'my online scores' menu
2. Click on a score
3. Click on another score before the first one loads

### Screenshots/Screen recordings

_No response_

### MuseScore Version

4.1

### Regression

Choose option...

### Operating system

Windows 10

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by Eism**
Resolves: #18558

### CODE REVIEW

File: src/project/internal/projectactionscontroller.cpp [hunk 1]
```diff
@@ -163,7 +163,7 @@ Ret ProjectActionsController::openProject(const ProjectFile& file)
     //! the events (like user click) can be executed and this method can be called several times,
     //! before the end of the current call.
     //! So we ignore all subsequent calls until the current one completes.
-    if (m_isProjectProcessing) {
+    if (m_isProjectProcessing || m_isProjectDownloading) {
```
Comment (cbjeukendrup) [inline src/project/internal/projectactionscontroller.cpp:166]: I think you need to do the same in `ProjectActionsController::newProject() `

**Review by cbjeukendrup — COMMENTED** [review]
I was about to create approximately the same PR because you had assigned me, but it's alright :)

**Comment by Eism** [comment]
I was just processing crash reports from Sentry and accidentally fixed this bug

**Comment by DmitryArefiev** [comment]
Tested #18558 on Win10, Mac13, LinuxUbuntu 22.04 - **FIXED**

**Review by RomanPudashkin — APPROVED** [review]

**Pull request merged**